include/grpc/impl/codegen/sync_posix.h

```diff
@@ -25,8 +25,26 @@
 
 #include <pthread.h>
 
+#ifdef GRPC_ASAN_ENABLED
+/* The member |leak_checker| is used to check whether there is memory leak
+ * that may be caused by upper layer logic which missing the |gpr_xx_destroy|
```

Comment:
Grammar nits: I'd suggest the following for the comment: 
"The member |leak_checker| is used to check whether there is a memory leak caused by upper layer logic that's missing the |gpr_xx_destroy| call to the object before freeing it."

Reply:
Thanks! I'll make a commitment to fix it later.

Reply:
done